pkg/apis/release/v1alpha1/types.go

```diff
@@ -208,15 +208,40 @@ type ReleasePayloadOverride struct {
 	Reason string `json:"reason,omitempty"`
 }
 
+// PayloadVerificationConfig specifies the configuration used to verify the ReleasePayload
+// +kubebuilder:validation:XValidation:rule="!has(oldSelf.payloadVerificationDataSource) || has(self.payloadVerificationDataSource)", message="PayloadVerificationDataSource is required once set"
```

Comment:
if payloadVerificationDataSource, can blockingJobs, informingJobs, or upgradejobs be set?  if not, let's get a rule to enforce that.

Reply:
The expectation is that the creator of the ReleasePayload populates any tests that are associated with the payload.  The underlying controllers will act accordingly and populate the respective pieces.  Any results will be populated by either the ProwJob Controller or the Legacy Controller.